src/station_cmd.cpp

```diff
@@ -864,6 +866,79 @@ static CommandCost CheckFlatLandAirport(AirportTileTableIterator tile_iter, DoCo
 	return cost;
 }
 
+static StringID GetDemolishStringForStationType(StationType type)
```

Comment:
Missing doxygen.

Reply:
Removed in favour of using the standard technique of trying clear the land, which will produce the correct error.